api/client.go

```diff
@@ -154,38 +158,8 @@ func patch[Req, Res any](client Client, path string, req *Req) (res *Res, err er
 }
 
 func delete(client Client, path string) error {
```

Comment:
Noticed this wasn't adding the common headers. Probably should. Could be its own PR